tests/test_systrace.py

```diff
@@ -55,14 +57,32 @@ def test_cpu_counting(self):
     def test_systrace_userspace(self):
         """Test parsing of userspace events"""
 
+        # Test a 'B' event (begin)
         trace = trappy.SysTrace("trace_sf.html")
         dfr = trace.tracing_mark_write.data_frame
-        self.assertTrue(dfr['__pid'].iloc[2], 7459)
-        self.assertTrue(dfr['__comm'].iloc[2], 'RenderThread')
-        self.assertTrue(dfr['pid'].iloc[2], 7459)
-        self.assertTrue(dfr['event'].iloc[2], 'B')
-        self.assertTrue(dfr['func'].iloc[2], 'notifyFramePending')
-        self.assertTrue(dfr['data'].iloc[-2], 'HW_VSYNC_0')
+        self.assertEquals(dfr['__pid'].iloc[2], 7591)
```

Comment:
Yes, probably. For now I think I will prefer to leave this as is since it involves an overhaul of this part of the test. For future test cases, will keep it in mind